### DISCUSSION TITLE
Confusing version.h file?

### DISCUSSION THREAD

**Discussion by gitfve** [mriscoc/Ender3V2S1#1050] [General]
Confusing version.h file?

This is in all version, why not update it on your new build?
Write the real Marlin build version and so on....

#define SHORT_BUILD_VERSION "**2.1.3** MRiscoC"
#define DETAILED_BUILD_VERSION SHORT_BUILD_VERSION " Ender3S1-F4, based on **bugfix-2.1.x**"

**Comment by mriscoc** [comment]
Because this firmware is not a Marlin version, it is build over Marlin bugfix-2.1.x. Current Marlin release is 2.1.2.1, the convention of Professional Firmware is add a 0.0.1 to the last released version. You can't get a Professional Firmware by compiling Marlin and only changing the configuration files.

**Comment by gitfve** [comment]
Just to understand it a little better.
Are you using the "Marlin Nightly Source, Work in progress - Updates for 2.1.x bugfix snapshot" as you source?

Why not use the latest stable marlin 2.1.2.1?

**Reply by mriscoc** [reply]
The firmware is mostly based on the Marlin Bugfix but some chosen PRs are merged if they fix some evident bug. If you look at the latest stable Marlin you will find that some important errors are not fixed and trying to merge the fix into the the stable branch is more difficult due to the changes in the Bugfix branch.

**Comment by gitfve** [comment]
Thanks:-)
